sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala

```diff
@@ -687,6 +691,40 @@ object DataSourceStrategy
     (nonconvertiblePredicates ++ unhandledPredicates, pushedFilters, handledFilters)
   }
 
+  protected[sql] def translateAggregate(aggregates: AggregateExpression): Option[AggregateFunc] = {
+
+    def columnAsString(e: Expression): String = e match {
+        case AttributeReference(name, _, _, _) => name
+        case Cast(child, _, _) => columnAsString(child)
+        case _ => ""
+    }
+
+    aggregates.aggregateFunction match {
+      case min: aggregate.Min =>
+        val colName = columnAsString(min.child)
```

Comment:
One suggestion: Can we reuse `PushableColumn` which is used by predicate pushdown to capture pushed column?